Nickel/Framework/Content/CardTraitManager.cs

```diff
@@ -175,45 +179,43 @@ public void AddCardTraitOverride(Card card, string uniqueName, bool overrideValu
 	{
 		if (this.SynthesizedVanillaEntries.Value.TryGetValue(uniqueName, out var vanillaEntry))
 		{
-			vanillaEntry.setOverride(card, overrideValue);
-			if (isPermanent)
-				vanillaEntry.setPermanent(card);
+			vanillaEntry.SetOverride(card, overrideValue);
+			vanillaEntry.SetPermanent(card, isPermanent);
```

Comment:
n.b. this change breaks the vanilla behavior that I intentionally wanted to mirror (and also mirror for custom traits below) when adding card trait overrides: Nothing ever sets `*OverrideIsPermanent = false`, so if you have some permanent override that something *would normally* temporarily write over, that override becomes permanent instead.